### PR TITLE
Add sympy tab to pl-variable-output

### PR DESCRIPTION
**Pull request by fmarrar**
Added Sympy tab to pl-variable-output.

### CODE REVIEW

File: elements/pl-drawing/mechanicsObjects.js [hunk 1]
```diff
@@ -2037,12 +2037,7 @@ mechanicsObjects.byType['pl-arc-dimensions'] = class extends PLDrawingBaseElemen
 
           if (obj.label) {
               let dt = obj.endAngle-obj.startAngle;
-              let t;
-              if (dt >= 0) {
-                  t = obj.startAngle + dt/2;
-              } else {
-                  t = obj.startAngle + (360+dt)/2;
-              }
+              let t = obj.startAngle + dt/2;
```
Comment (nicknytko) [inline elements/pl-drawing/mechanicsObjects.js:2040]: Are these changes relevant to the variable output?

Comment (fmarrar) [reply elements/pl-drawing/mechanicsObjects.js:2040]: No. This is just for label offset in pl-arc-dimensions where in certain cases, the label offset is completely incorrect due to the else statement.

Comment (mwest1066) [reply elements/pl-drawing/mechanicsObjects.js:2040]: @fmarrar it looks like you tangled this branch together with the one from #4788

Comment (mwest1066) [reply elements/pl-drawing/mechanicsObjects.js:2040]: When you made the `add-sympy-tab` branch, it looks like you forgot to switch back to `master` first. That is, you were still on the `fix-arc-dimensions-label` branch when you made the `add-sympy-tab` branch, so `add-sympy-tab` includes all the changes from `fix-arc-dimensions-label`. To fix this, you should do:
```
git checkout master
git pull
git checkout add-sympy-tab
git rebase --onto master fix-arc-dimensions-label add-sympy-tab
git push --force origin add-sympy-tab
```

Comment (fmarrar) [reply elements/pl-drawing/mechanicsObjects.js:2040]: Extremely sorry about that. I did -u when I pushed so I thought it would not conflict with the other branch.

Comment (mwest1066) [reply elements/pl-drawing/mechanicsObjects.js:2040]: No worries. The problem isn't with the push, the problem was with the branches themselves on your local machine. But the above commands will fix it.

Comment (fmarrar) [reply elements/pl-drawing/mechanicsObjects.js]: Just did that. Again, incredibly sorry about that.

Comment (mwest1066) [reply elements/pl-drawing/mechanicsObjects.js]: No worries at all, we've all messed up with git before 😄

**Comment by mwest1066** [comment]
@nicknytko does this (and #4788) look sensible to you?

**Review by coatless — APPROVED** [review]
Not sure why I was tagged on this review, but the changes look good. I'll make this as approved, but I have two notes:

1. the `SymPy` 1d vector would show as a column-based vector instead of row-based vector. 

https://docs.sympy.org/latest/tutorial/matrices.html

2. `elements.md` in `docs/` needs to be updated to include the new options.

**Comment by nicknytko** [comment]
Minor note, do we want to add some import to the sympy tab like `from sympy import *` to match the numpy tab?

image

**Comment by fmarrar** [comment]
> Minor note, do we want to add some import to the sympy tab like `from sympy import *` to match the numpy tab?
> 
> image

Good idea! Just did that.

File: elements/pl-variable-output/pl-variable-output.mustache [hunk 2]
```diff
@@ -14,6 +14,8 @@
             {{#show_python}}<li class="nav-item" role="presentation"><a class="nav-link{{#active_tab_python}} active{{/active_tab_python}}" href="#python-{{uuid}}" aria-controls="python-{{uuid}}" role="tab" data-toggle="pill">Python</a></li>{{/show_python}}
 
             {{#show_r}}<li class="nav-item" role="presentation"><a class="nav-link{{#active_tab_r}} active{{/active_tab_r}}" href="#r-{{uuid}}" aria-controls="r-{{uuid}}" role="tab" data-toggle="pill">R</a></li>{{/show_r}}
+
+            {{#show_sympy}}<li class="nav-item" role="presentation"><a class="nav-link{{#active_tab_sympy}} active{{/active_tab_sympy}}" href="#sympy-{{uuid}}" aria-controls="sympy-{{uuid}}" role="tab" data-toggle="pill">Sympy</a></li>{{/show_sympy}}
```
Comment (mwest1066) [inline elements/pl-variable-output/pl-variable-output.mustache:18]: Use the preferred capitalization from https://www.sympy.org/en/index.html
```suggestion
            {{#show_sympy}}<li class="nav-item" role="presentation"><a class="nav-link{{#active_tab_sympy}} active{{/active_tab_sympy}}" href="#sympy-{{uuid}}" aria-controls="sympy-{{uuid}}" role="tab" data-toggle="pill">SymPy</a></li>{{/show_sympy}}
```

**Review by mwest1066 — APPROVED** [review]
